org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -53,6 +53,11 @@
 	 */
 	public static final String JAVA_FORMAT_ENABLED_KEY = "java.format.enabled";
 
+	/**
+	 * Preference key to enable/disable auto organize imports
+	 */
+	public static final String JAVA_SAVE_ACTION_AUTO_ORGANIZE_IMPORTS_KEY = "java.saveAction.organizeImports";
```

Comment:
java.saveActions.organizeImports // saveActions plural, since we expect more will come later